test/regress_http.c

```diff
@@ -2844,9 +2932,8 @@ http_parse_query_str_flags_test(void *ptr)
 static void
 http_parse_uri_test(void *ptr)
 {
-	const int nonconform = (ptr != NULL);
-	const unsigned parse_flags =
-	    nonconform ? EVHTTP_URI_NONCONFORMANT : 0;
```

Comment:
Loose `EVHTTP_URI_NONCONFORMANT`, the problem that tests does not found, because the test was wrong